src/main/java/org/opensha/sha/earthquake/faultSysSolution/erf/BaseFaultSystemSolutionERF.java

```diff
@@ -313,14 +313,21 @@ protected void updateHookBeforeFaultSourceBuild() {
 	}
 	
 	@Override
-	public void updateForecast() {
+	public void updateForecast() throws NullPointerException {
 		
 		if (D) System.out.println("Updating forecast");
 		long runTime = System.currentTimeMillis();
 		
+		if (faultSysSolution == null) {
+			if (D) System.out.println("Failed to update forecast, faultSysSolution == null.");
+			throw new NullPointerException(
+					"Failed to update forecast. Fault system solution is unavailable. "
+					+ "Ensure the ERF is provided prior to computation.");
+		}
+		
```

Comment:
shouldn't this be after the `if (fileParamChanged) { readFAultSystemSolutionFromFile(); }` code? As currently written, I don't think the file based version of the code could ever work (you can test by selecting "Fault System Solution ERF" and browsing for a solution file).